potpie/context-engine/src/potpie_context_engine/adapters/outbound/resources/parsers/docling_convert.py

```
@@ -0,0 +1,157 @@
+    provenance_map: dict = {}
+    section_ocr: dict[str, list[str]] = {}
+
+    try:
+        parsed_sections, elements, provenance_map, section_ocr = (
+            build_sections_from_docling(
+                document,
+                chunk_target=chunk_target,
+                artifacts_dir=artifacts_dir,
+            )
+        )
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)